server/environment-variables/local.json

```diff
@@ -55,7 +55,8 @@
       "security": [
         "bdavidson@cynerge.com"
       ],
-      "smtp_server": "smtp.sendgrid.net",
+      "port": 25,
+      "smtp_server": "mailproxy1.usda.gov",
```

Comment:
I think I see what you were saying now about some stuff being pulled in from another branch @Dmac26! This looks out of scope too.